### PR TITLE
jsconfig: Add further jsconfig excludes according to latest Microsoft suggestions.

### PR DESCRIPTION
**Pull request by kwight**
I'm still seeing "exclude large folders" warnings in VS Code with our current `jsconfig.json`. I noticed in [recent Microsoft advice](https://github.com/Microsoft/vscode/issues/31711#issuecomment-344795394) this added `node_modules` rule; I'm the first to admit I don't see how the addition of this rule would make a difference in Calypso, but it takes care of my warnings. ¯\_(ツ)_/¯

**Testing**
* Do a fresh checkout of Calypso.
* Open VS Code within Calypso, and open a file or two; you should see a prompt to exclude large files.
* Switch to this branch, and do the same; you shouldn't see the warning.

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -5,12 +5,17 @@
...
+			]
 		}
 	},
 	"exclude": [
 		"node_modules",
+		"**/node_modules/*",
 		"public",
 		"build"
 	]
-}
```
Comment (samouri) [inline jsconfig.json]: can you keep the newline

Comment (kwight) [reply jsconfig.json]: Prettier says no; do we need to update something in our Prettier fork?

Comment (samouri) [reply jsconfig.json]: i didn't think we were prettifying `.json` files :/.
It doesn't really matter one way or the other. feel free to ship without keeping newline

Comment (samouri) [reply jsconfig.json]: also, sidenote:  you can always skip auto-prettification by doing:
```bash
git commit -n 
```

**Comment by blowery** [comment]
Hmmm, still seeing warnings here.. Anyone else?

**Comment by samouri** [comment]
Yep, im seeing them too

**Comment by kwight** [comment]
Yep, me three :/ I don't think there's a need to revert this, but it's still an obvious issue and irritant.

**Pull request merged**